readthedocs/api/v2/mixins.py

```diff
@@ -37,10 +37,10 @@ def _get_cache_tags(self):
             version = self._get_version()
             tags = [
                 project.slug,
-                f'{project.slug}-{version.slug}',
+                f'{project.slug}:{version.slug}',
```

Comment:
Not related to this PR, but, are we caching resource API calls? I guess we are not invalidating the cache properly for all the cases. Example: a new `Version` object is created, or a new `EnvironmentVariable` or similar.

Reply:
Cache is done on proxied APIs only.